src/applications/discharge-wizard/components/AirForcePortalLink.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const AirForcePortalLink = () => (
+  <section className="vads-u-margin-bottom--9">
+    <a
+      className="airForce-portal-link vads-u-display--flex vads-u-align-items--center vads-u-text-decoration--none"
+      href="https://afrba-portal.cce.af.mil/#portal"
```

Comment:
I think we need rel="noreferrer noopener" here 🙂 also do you want this to be a target="_blank"?

Reply:
I'll add the `rel="noreferrer noopener"`. That was to tell the other website Analytics that it's direct traffic? My memory does not serve me well on why we use the attribute.

As far as opening a new tab, a11y team requested that it doesn't open a tab and we navigate to the new portal.

Reply:
Oh this explanation helps a bit. 
https://stackoverflow.com/a/50709724

Reply:
Looks like we are gtg for links not opening in new tabs!